git-helper.js

```diff
@@ -34,19 +34,19 @@ exec('git tag --list', options)
 
 // Tags the repository at the current state
 const tag = (tag, message, push, options) =>
-exec(`git tag -a ${tag} -m "${message}"`, options)
+exec(`git tag -a ${tag} -m "${message}" ${options.branch || ''}`, options)
 .then(() => {
   if (push) {
     return exec('git push --tags', options);
   }
 }).then(() => tag);
 
 
-// Gets the current branch
-const branch = options => exec('git rev-parse --abbrev-ref HEAD', options);
+// Gets the provided branch or the current branch
+const branch = options => options.branch || exec('git rev-parse --abbrev-ref HEAD', options);
 
 
-// Checks to see if the current local repository is clean
+// Checks to see if the repository is clean
```

Comment:
"Checks to see if the current local state of the repository is in sync with its remote"